### ISSUE TITLE
USER ISSUE: Enter wrong password in login screen leaves wrong pop-up block.

### ISSUE THREAD

**Issue by BlazeGuy2** [StrangeLoopGames/EcoIssues#9831]
**Version:** 0.7.8.0 beta master-03df139f debug

**Steps to Reproduce:**
Login to Eco w/ wrong password.

**Expected behavior:**
Error msg should say Entered Incorrect Username/Password

**Actual behavior:**
The following error message popped up:
```
Error or otherwise informative message goes here.
```n
**Do you have mods installed? Does issue happen when no mods are installed?:**
The following error message popped up:
```
Error or otherwise informative message goes here.
```

**Comment by D3nnis3n** [comment]
As NONE of the issues that had to do with "Error or otherwise informative message goes here." have been fixed in the latest staging build i assumed this was one of those bugs not merged into staging branch and did not report. I did report the issue in conjunction with https://github.com/StrangeLoopGames/EcoIssues/issues/9800, though.

Seems like i also have to keep track what happens with my issues for good QA, so i know if it's expected to be in the staging version or not. No GitHub professional, though, could take some time until i fiddled that out.
